control_ni_switch

```diff
@@ -12,5 +12,5 @@ XB-UserVisible: yes
 Section: Add-Ons
 XB-DisplayVersion: {display_version}
 XB-DisplayName: NI-SWITCH for VeriStand {veristand_version}
-Depends: {ni-veristand-{labview_version}} (>= {labview_short_version}.0.0)
```

Comment:
NI-SWITCH has a complicated runtime package layout, with some of the runtime hardware-enablement packages being optional. Rather than enumerate all those dependencies here, just depend on the top level package and let NI-SWITCH sort things out.